assets/js/util.js

```diff
@@ -9,6 +9,9 @@ var SPACE_KEY_CODE = 32, ENTER_KEY_CODE = 13,
 
 var TEXTAREA_AUTO_RESIZE_MINIMUM_WIDTH = 768;
 
+var amountScrolled = 300; /*:: amount by which user must scroll to view the button */
+var transitionTime = 600; /*:: time taken for the transition to move webpage to top */
```

Comment:
@sushain97 by same naming style, do we mean I need to change to something like:
var AMOUNT_SCROLLED = 300;
var TRANSITION_TIME = 600;??

> Move these to be part of the var above,

Means I have to write these above the TEXT_AUTO_RESIZE_MINIMUM_WIDTH right?
I would remove the variable names.